sql/src/test/java/io/crate/analyze/CopyAnalyzerTest.java

```diff
@@ -112,6 +108,27 @@ public void testCopyFromParameter() throws Exception {
         assertThat(BytesRefs.toString(value), is(path));
     }
 
+    @Test
+    public void testCopyFromWithInputFormatJson() {
+        CopyFromAnalyzedStatement analysis = e.analyze("copy users from '/some/distant/file.ext' with (format='json')");
+        assertThat(analysis.table().ident(), is(USER_TABLE_IDENT));
+        assertThat(analysis.inputFormat(), is(WriterProjection.InputFormat.JSON));
```

Comment:
Check the path here?

Reply:
Not sure I see what you're asking

Reply:
I meant to say, how about checking the path from the `copy users from <path>` here?